pkg/cmd/task/task_exec.go

```diff
@@ -0,0 +1,62 @@
+package task
+
+import (
+	"os"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/dcos/dcos-core-cli/pkg/mesos"
+	"github.com/mesos/mesos-go/api/v1/lib/httpcli/httpagent"
+	"github.com/spf13/cobra"
+)
+
+func newCmdTaskExec(ctx api.Context) *cobra.Command {
+	var interactive, tty bool
+	var user string
+
+	cmd := &cobra.Command{
+		Use:   "exec <task> <cmd> [<args>...]",
+		Short: "Launch a process (<cmd>) inside of a container for a task (<task>).",
+		Args:  cobra.MinimumNArgs(2),
```

Comment:
TODO: we should disable flag parsing after the `<task>` argument, not to have cobra fail or mix flags of the exec command with potential flags of the command to exec.

Reply:
https://jira.mesosphere.com/browse/DCOS_OSS-5283